postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -66,7 +66,7 @@ instances:
     #
     # ssl: 'false'
 
-    ## @param query_timeout - integer - optional
+    ## @param query_timeout - integer - optional - default: 1000
```

Comment:
Not sure why this and `dbname` have changed? You might need to `ddev validate config --sync`.

Reply:
Ah, turns out it was my ddev that was out of date. I re-ran the config sync and it matches master again.